build/import/Packages.targets

```diff
@@ -102,12 +102,12 @@
 
     <!-- Tests -->
     <PackageReference Update="Moq"                                                                    Version="4.12.0"/>
-    <PackageReference Update="xunit"                                                                  Version="2.4.1" />
-    <PackageReference Update="xunit.assert"                                                           Version="2.4.1" />
-    <PackageReference Update="xunit.extensibility.core"                                               Version="2.4.1" />
+    <PackageReference Update="xunit"                                                                  Version="2.4.2-pre.13" />
+    <PackageReference Update="xunit.assert"                                                           Version="2.4.2-pre.13" />
+    <PackageReference Update="xunit.extensibility.core"                                               Version="2.4.2-pre.13" />
```

Comment:
Should we pull this version out into a property? That is, will we always want to update these three packages in unison?